app/views/users/new.html.erb

```diff
@@ -0,0 +1,42 @@
+<div style="text-align:center;"> 
+<h1>Sign up</h1>
+</div>
+<div class="w-25 container">
+<%= form_with model: @user do |form| %>
+  <%if @user.errors.any? %>
+    <div class="alert alert-danger">
+    <% @user.errors.full_messages.each do |message| %>
+      <div><%= message %></div>
+    <% end %>
+    </div>
+  <% end %>
```

Comment:
I think this similar logic is repeating in multiple files. We can extract this in a shared partial.